plugins/labhub.py

```diff
@@ -122,6 +126,10 @@ def invite_cmd(self, msg, match):
         """
         invitee = match.group(1)
         inviter = msg.frm.nick
+        if not inviter:
+            self.send(msg.frm, 'ERROR: The above command cannot be '
```

Comment:
errors should be yielded

Reply:
Got it :+1:

Reply:
@abhishalya, since `members_only` is a command filter and it is called before the desired bot command method is called. So you don't need to check for `nick` in every command method.

Reply:
@nvzard Yeah, I knew that. I suggested that on zulip, but since no one responded I appended the same code to all. Will do this change :+1: